webapp/tests/karma/ts/services/rules-engine.service.spec.ts

```diff
@@ -457,7 +462,7 @@ describe('RulesEngineService', () => {
 
     expect(rulesEngineCoreStubs.fetchTasksFor.callCount).to.eq(1);
     expect(rulesEngineCoreStubs.fetchTargets.callCount).to.eq(1);
-    expect(telemetryService.record.callCount).to.equal(3);
+    expect(telemetryService.record.callCount).to.equal(5);
```

Comment:
How come this changed? Which additional entries does telemetry record?

Reply:
Same than the other case about the promise.